LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2005-2024, NumPy Developers.
+Copyright (c) 2005-2025, NumPy Developers.
```

Comment:
I think removing the end year is more in line with current recommendations, see the discussion on cpython starting [here](https://github.com/python/cpython/issues/126133#issuecomment-2460824052) where the legal team OKs removing the end year.
```suggestion
Copyright (c) 2005, NumPy Developers.
```

Reply:
It's really not clear to me that they are. FWIW, I'm happy with leaving any copyright notices in each individual file with the first year of original authorship (and never updating them again), but the maintenance burden of the keeping the `LICENSE` notice correct each year seems trivial to me.

Reply:
> It's really not clear to me that they are.

To clarify, I mean that the [cited email from PSF's legal](https://github.com/python/cpython/issues/126133#issuecomment-2460824052) says that "both of the formats you propose that include the year would be fine", but they never posted the email this was in response to. I have no idea what those two proposed formats are or which "the year" they were referring to.

Reply:
Based on the recommendations from the PSF legal team and your feedback, using only the start year for the copyright is a suitable and current approach. This not only aligns with legal guidelines but also reduces the maintenance burden.

If you agree with removing the end year from the copyright notice, please let me know so I can apply this change on this PR.
I look forward to the team's consensus on this matter.

Reply:
> but they never posted the email this was in response to ...

Right but they did [remove the end years](https://github.com/python/cpython/pull/126236/files) from CPython and [documented that policy](https://devguide.python.org/getting-started/pull-request-lifecycle/#copyrights)

Reply:
> The only burden it ever caused are these discussions trying to change our policy from just merging the inevitable PR each year.

OK, let's put this in as is.